src/partials/header.html

```diff
@@ -1 +1,180 @@
+            </li>
+            <li class="header-nav-item">
+              <a class="header-nav-link-five" href="#leave-section"
+                >Contact Us</a
+              >
+            </li>
+          </ul>
+        </nav>
+        <ul class="header-socials list">
+          <li class="header-socials-item">
+            <a
+              class="header-socials-link link"
+              href="https://facebook.com"
+              target="_blank"
+              rel="noopener noreferrer"
+              aria-label="Facebook link"
+            >
+              <svg class="header-socials-icon" width="28" height="28">
+                <use href="./images/icons.svg#icon-facebook"></use>
+              </svg>
+            </a>
+          </li>
+          <li class="header-socials-item">
+            <a
+              class="header-socials-link link"
+              href="https://instagram.com"
+              target="_blank"
+              rel="noopener noreferrer"
+              aria-label="Instagram link"
+            >
+              <svg class="header-socials-icon" width="28" height="28">
+                <use href="./images/icons.svg#icon-instagram"></use>
+              </svg>
+            </a>
+          </li>
+          <li class="header-socials-item">
+            <a
+              class="header-socials-link link"
+              href="https://youtube.com"
+              target="_blank"
+              rel="noopener noreferrer"
+              aria-label="youtube link"
+            >
+              <svg class="header-socials-icon" width="24" height="24">
+                <use href="./images/icons.svg#icon-youtube"></use>
+              </svg>
+            </a>
+          </li>
+        </ul>
+        <!-- Mobile Menu -->
+
+        <button
+          class="menu-open-button js-open-menu"
+          type="button"
+          aria-expanded="false"
+          aria-controls="mobile-menu"
+          data-menu-open-btn
+        >
+          <svg
+            class="menu-open-icon"
+            width="32"
+            height="23"
+            stroke="var(--color-black)"
+            aria-controls="mobile-menu"
+            aria-expanded="false"
+            role="img"
+          >
+            <use
+              class="menu-open-use"
+              xlink:href="./images/icons.svg#icon-Nav"
+            ></use>
+          </svg>
+        </button>
+      </div>
+      <!-- Mobile Menu container -->
+      <div class="backdrop is-hidden" data-menu-backdrop>
+        <div class="menu-container js-menu-container" id="mobile-menu">
+          <button
+            type="button"
+            class="menu-close-button js-close-menu"
+            data-menu-close-btn
+          >
+            <svg class="menu-close-icon" width="32" height="32">
+              <use href="./images/icons.svg#icon-Nav"></use>
+            </svg>
+          </button>
+
+          <!-- Mobile Menu Navigation  -->
+
+          <nav class="menu-nav">
```

Comment:
mobile navigation should not be surrounded with `nav` tag
use `div` instead

Reply:
+